pypdf instead of pypdf4, natsort dep setuptools added (was distutils)

drupchen opened the pull request:
pypdf4 is not supported anymore, but pypdf is active once again. distutils is deprecated since pypthon3.10, but included in setuptools.

drupchen committed:
pypdf instead of pypdf4, natsort dep setuptools added (was distutils)

pypdf4 is not supported anymore, but pypdf is active once again. distutils is deprecated since pypthon3.10, but included in setuptools.

drupchen committed:
split output in odd and even pages for ease of printing

drupchen committed:
if debug, draws cutting lines in red

eroux commented:
Hi @drupchen ! Just FYI, a couple of years ago I looked at PDF libraries in Python and they were all having a problem: they were reencoding images when including them, thus lowering the quality. I spent quite some time making fpdf2 not do that so that's whay we use at BDRC now. I think it would make sense to use it in this project too

drupchen committed:
add debug mode + printer adjustment

debug adds lines at vertical boundaries of pages, adjustment lowers images on the page to account for printer border offset

drupchen committed:
add debug mode + printer adjustment

debug adds lines at vertical boundaries of pages, adjustment lowers images on the page to account for printer border offset

Yoesel commented:
respected Sir/Madam.
I am going request with utmost respect you. please guide me how to download pecha printer.
